main/mafia.hs

```diff
@@ -144,7 +145,12 @@ run = \case
     mafiaScript path args
 
 parser :: Parser MafiaCommand
-parser = subparser . mconcat $ commands
+parser =
+  subparser (mconcat commands) <|> pScript
+
+pScript :: Parser MafiaCommand
+pScript =
+  MafiaScript <$> pScriptPath <*> many pScriptArgs
```

Comment:
yeah, that's right, just for the help text, although there's an optparse PR up at the moment which will allow us to avoid the hack at the top (see link in the code)